G4_DailyReport/src/main/java/com/example/G4_DailyReport/controller/manager/ReportController.java

```diff
@@ -28,12 +31,16 @@
 public class ReportController {
 
     private final ReportService reportService;
+    //for test only
+    private final ReportRepository rerportRepo;
```

Comment:
```suggestion
    private ReportService reportService;
    //for test only
    private ReportRepository rerportRepo;
```
cần j final thanh niên

ngoài ra, ở controller thì call service nhé, chư sko phải repository đâu gái

Reply:
dạ final là để em dùng cái @RequiredArgsConstructor nó bơm cho bọn final mà không cần autowired á chị
còn cái repo đó thì đúng là phải bỏ đi á.